### PR TITLE
Feature make website responsive

### PR DESCRIPTION
**Pull request by JohnL17**

### CODE REVIEW

File: app/static/js/scroll_arrow.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+console.log("Hello");
+$(".arrow-wrap").click(function() {
+    console.log($(".how-it-works"))
```
Comment (czosel) [inline app/static/js/scroll_arrow.js]: Better remove the `console.log` statements

File: app/templates/home.html [hunk 2]
```diff
@@ -5,54 +5,127 @@
...
+            </p>
+        </div>
+        <div class="col-xs-12 col-md-8">
+            <img class="img-responsive c3" src="{% static 'images/c3.png' %}" alt="Web GUI">
+        </div>
     </div>
+    <div class="ghost-button-div">
+        <button class="ghost-button" onclick="location.href='/registration'" type="button">Sign up now!</button>
+    </div>
+    <!--<div id="carousel" class="carousel slide" data-ride="carousel" data-interval="9000">-->
```
Comment (czosel) [inline app/templates/home.html]: Please remove the escaped HTML

File: app/static/js/scroll_arrow.js (hunk 1 above)
Comment (JohnL17) [inline app/static/js/scroll_arrow.js]: sorry I forgot that.

File: app/templates/home.html (hunk 2 above)
Comment (JohnL17) [inline app/templates/home.html]: It's removed now.

**Pull request merged**